example/src/main/java/io/netty/example/factorial/FactorialClientInitializer.java

```diff
@@ -43,7 +43,7 @@ public void initChannel(SocketChannel ch) {
 
         // Enable stream compression (you can remove these two if unnecessary)
         pipeline.addLast(ZlibCodecFactory.newZlibEncoder(ZlibWrapper.GZIP));
-        pipeline.addLast(ZlibCodecFactory.newZlibDecoder(ZlibWrapper.GZIP));
+        pipeline.addLast(ZlibCodecFactory.newZlibDecoder(ZlibWrapper.GZIP, 65536));
```

Comment:
Consider using a static constant to make it easier for readers of all examples to understand what this number means

Reply:
Moved to a static constant